resources/views/applicant/strategic_response_application/step_01/questions.html.twig

```diff
@@ -4,135 +4,132 @@
+                        </div>
+                    </div>
+                    <div class="box full">
+                        <div class="form__input-wrapper--date">
+                            <label class="form__label" for="director_title">
+                                {{ application_template.strategic_response.director_title_label }}
+                            </label>
+                            <input class="form__input" id="director_title" name="director_title" required type="text"/>
+                        </div>
+                    </div>
+                    <div class="box full">
+                        <div class="form__input-wrapper--date">
+                            <label class="form__label" for="director_email">
+                                {{ application_template.strategic_response.director_email_label }}
+                            </label>
+                            <input class="form__input" id="director_email" name="director_email" required type="email"/>
+                        </div>
+                    </div>
+                    <p>
+                        {{ application_template.strategic_response.reference_copy }}
+                    </p>
+                    <div class="box full">
+                        <div class="form__input-wrapper--date">
+                            <label class="form__label" for="reference_name">
+                                {{ application_template.strategic_response.reference_name_label }}
+                            </label>
+                            <input class="form__input" id="reference_name" name="reference_name" required type="text"/>
+                        </div>
+                    </div>
+                    <div class="box full">
+                        <div class="form__input-wrapper--date">
+                            <label class="form__label" for="reference_title">
+                                {{ application_template.strategic_response.reference_title_label }}
+                            </label>
+                            <input class="form__input" id="reference_title" name="reference_title" required type="text"/>
+                        </div>
+                    </div>
+                    <div class="box full">
+                        <div class="form__input-wrapper--date">
+                            <label class="form__label" for="reference_email">
+                                {{ application_template.strategic_response.reference_email_label }}
+                            </label>
+                            <input class="form__input" id="reference_email" name="reference_email" required type="email"/>
+                        </div>
+                    </div>
+                </div>
             </div>
-
         </div>
-
         <h3>
             {{ application_template.question_title }}
         </h3>
```

Comment:
I think we can actually remove this title ("My Fit") and just leave everything under "Required Information".

Reply:
Done.